src/account/AccountItem.js

```diff
@@ -42,29 +40,75 @@ export default class AccountItem extends React.PureComponent {
   };
 
   handleSelect = () =>
-    this.props.onSelect(this.props.index);
+  this.props.onSelect(this.props.index);
 
   handleRemove = () =>
-    this.props.onRemove(this.props.index);
+  this.props.onRemove(this.props.index);
 
   render() {
-    const { email, realm } = this.props;
+    const { email, realm, index } = this.props;
 
     return (
       <Touchable style={styles.wrapper} onPress={this.handleSelect}>
-        <View style={styles.accountItem}>
+        <View style={[styles.accountItem, this.getDynamicStyles(index)]}>
```

Comment:
Have a look at how these are implemented for StreamItem. They work almost exactly as @neerajwahi describes.